tcms/report/views.py

```diff
@@ -442,10 +442,10 @@ def _report_data_context(self):
                 self._data.generate_status_matrix(build_ids)
             )
 
-            status_ids = (TestCaseRunStatus.id_failed(),)
+            status_ids = (TestCaseRunStatus.status_to_id(TestCaseRunStatus.FAILED),)
```

Comment:
I think it's best to directly use `objects.get()` with the class attribute constants (w/o the extra upcase()) and without the extra try-except coming from `status_to_id()`. This way we will get failures when there are no such objects so they can be fixed faster.

Note: searching by name can fail when the names are translated at the DB level.